godtools/TractClasses/TractProperties.swift

```diff
@@ -13,17 +13,21 @@ class TractProperties: XMLNode {
     var primaryColor = GTAppDefaultStyle.primaryColor.getRGBAColor()
     var primaryTextColor = GTAppDefaultStyle.primaryTextColorString.getRGBAColor()
     var textColor = GTAppDefaultStyle.textColorString.getRGBAColor()
+    var textScale: Int = 1
+    var textSize: Int = 18
     
     required override init() {
         super.init()
     }
     
     override func getProperties() -> [String] {
         defineProperties()
-        return ["primaryColor", "primaryTextColor", "textColor"] + self.properties
+        return ["primaryColor", "primaryTextColor", "textColor", "textScale", "textSize"] + self.properties
```

Comment:
we should never expect a textSize to be set as a property. we define the defaults and they can be scaled.